### PR TITLE
moved multiqc to lib/MIP/Program/Qc and created test

### PR DESCRIPTION
**Pull request by northwestwitch**
Tested on Rasta

### CODE REVIEW

File: lib/MIP/Program/Qc/Multiqc.pm [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+##           : $outdir_path            => Outdir path
+##           : $stderrfile_path        => Stderrfile path
+##           : $stdoutfile_path        => Stdoutfile path
+##           : $FILEHANDLE             => Filehandle to write to
+##           : $stderrfile_path_append => Append stderr info to file
+##           : $force                  => Force overwrite of output files
+
+    my ($arg_href) = @_;
+
+    ## Default(s)
```
Comment (henrikstranneheim) [inline lib/MIP/Program/Qc/Multiqc.pm]: Move default section to above $tmpl

File: lib/MIP/Program/Qc/Multiqc.pm [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    }
+
+    ## Outdir
+    if ($outdir_path) {
+
+        push @commands, q{--outdir} . $SPACE . $outdir_path;
+    }
+
+    ## Indir
+    if ($indir_path) {
```
Comment (henrikstranneheim) [inline lib/MIP/Program/Qc/Multiqc.pm]: Required so remove if

File: t/multiqc.t [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+        expected_output => $function_base_command,
+    },
+);
+
+## Can be duplicated with %base_argument and/or %specific_argument
+## to enable testing of each individual argument
+my %required_argument = (
+    indir_path => {
+        input           => catfile(qw{ path_to_in_dir infile }),
+        expected_output => q{},
```
Comment (henrikstranneheim) [inline t/multiqc.t]: You should expect the indir_path back

File: t/multiqc.t [hunk 4]
```diff
@@ -126,7 +126,7 @@ my %base_argument = (
 my %required_argument = (
     indir_path => {
         input           => catfile(qw{ path_to_in_dir infile }),
-        expected_output => q{},
+        expected_output => q{path_to_in_dir/infile},
```
Comment (henrikstranneheim) [inline t/multiqc.t]: use catfile otherwise it is OS dependent

Comment (northwestwitch) [reply t/multiqc.t]: Ah yes, true!

**Comment by henrikstranneheim** [comment]
👍

**Pull request merged**